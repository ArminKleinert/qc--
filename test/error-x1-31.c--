const     true = 1 == 1;

foo() {
    return <1 / true>;
}
