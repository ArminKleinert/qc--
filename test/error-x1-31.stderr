Character 42-60 Error: <x/y> illegal values
